pkg/azureclients/fileclient/azure_fileclient.go

```diff
@@ -119,14 +119,14 @@ func (c *Client) CreateFileShare(ctx context.Context, resourceGroupName, account
 	}
 	mc.Observe(rerr)
 
-	return err
+	return FileShare, err
 }
 
 // DeleteFileShare deletes a file share
-func (c *Client) DeleteFileShare(ctx context.Context, resourceGroupName, accountName, name string) error {
+func (c *Client) DeleteFileShare(ctx context.Context, resourceGroupName, accountName, name, xMsSnapshot string) error {
```

Comment:
add comments for `xMsSnapshot` parameter, e.g. what's the available values, you could copy the original comments from go sdk api

Reply:
ok, it's added.